storage_controller/src/http.rs

```diff
@@ -690,7 +690,8 @@ async fn handle_node_list(req: Request<Body>) -> Result<Response<Body>, ApiError
     };
 
     let state = get_state(&req);
-    let nodes = state.service.node_list().await?;
+    let mut nodes = state.service.node_list().await?;
```

Comment:
Deterministic order to enable the storcon CLI test assertions to run reliably with multiple nodes

Reply:
Maybe add this as a comment instead.